TASVideos.WikiEngine/NewParser.cs

```diff
@@ -856,7 +851,7 @@ private void ParseStartLine()
 			var e = new Element(_index, "code");
 			if (lang != "")
 			{
-				e.Attributes["class"] = "language-" + PrismNames.FixLanguage(lang);
+				e.Attributes["class"] = string.Concat("language-", PrismNames.FixLanguage(lang));
```

Comment:
Does this specific change gain us anything?  Seems like it's replacing `string + string` with `string.Concat(string, string)` which I would expect to be essentially identical.

Reply:
They are identical, they compile to the same code in the end (+ operator calls string.concat if both strings are not constant).